languages/translations/en.json

```diff
@@ -50,6 +50,11 @@
                 "destination": "Take the ferry towards {destination}"
             }
         },
+        "phrase": {
+            "twolinkedbydistance": "{instruction_one} then in {distance} {instruction_two}",
+            "twolinked": "{instruction_one} then {instruction_two}",
+            "oneindistance": "In {distance}, {instruction_one}"
```

Comment:
Should there be a comma here? "In 100 meters, turn right" or "In 100 meters turn right"

Reply:
I think that if you can drop the introductory clause it should be separated from the rest of the sentence by a comma. At least, that is how I remember @lyzidiamond explaining these things to me a few months back, but I'm pretty bad at grammar so I could very well be wrong here.

Reply:
Exactly right @mcwhittemore! "In 100 meters, turn right" is the correct structure.